developer-docs-site/docs/sdks/ts-sdk/sdk-core-layer.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Typescript SDK Core Layer"
+slug: "typescript-sdk-core-layer"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The core SDK layer exposes the functionalities needed by most applications:
+
+- Key generation
+- Transaction signing and submission
+- Transaction status querying
+- Information retrieval techniques
+
+### BCS Library
+
+The BCS serialization format is a key component of the Aptos blockchain ecosystem, providing a compact and efficient way to encode data on the blockchain. Its extensibility and support for custom data types make it a valuable tool for building blockchain applications on the Aptos network.
+BCS is a binary encoding format that is designed to be compact and efficient, making it ideal for use in blockchain applications where storage and bandwidth are at a premium. It is also designed to be extensible, allowing developers to define custom data types and structures.
+
+The TypeScript SDK implements a [Binary Canonical Serialization](https://docs.rs/bcs/latest/bcs/) (BCS) library for transaction signing and submission. The Aptos blockchain uses BCS for data serialization and deserialization. See the [Aptos SDK BCS](https://github.com/aptos-labs/aptos-core/tree/main/ecosystem/typescript/sdk/src/bcs).
```

Comment:
If the audience is SDK user. I wonder if they really need to know BCS from their side?

Reply:
I agree. I do think BCS needs to be mentioned as the SDK accepts BCS serialized transactions (although I would argue that the end user shouldn't provide a bcs serialized argument, but a argument and its type and the sdk should serialize it for them)